manifests/worker.pp

```diff
@@ -171,4 +179,12 @@
     ensure => $docker_ensure,
     dns    => $advertise_addr,
   }
+
+  if $xylem_backend {
```

Comment:
I'm pretty sure we do somewhere.. just can't think of where off-hand. Agree that this is cleaner.